extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -5,83 +5,16 @@
 
 import * as vscode from 'vscode';
 import * as constants from '../common/constants';
-import { AppSettingType, IDeployProfile, ILocalDbSetting } from '../models/deploy/deployProfile';
+import { AppSettingType, IDeployAppIntegrationProfile, IDeployProfile, ILocalDbSetting } from '../models/deploy/deployProfile';
 import { Project } from '../models/project';
-import * as generator from 'generate-password';
 import { getPublishDatabaseSettings } from './publishDatabaseQuickpick';
 import * as path from 'path';
 import * as fse from 'fs-extra';
 
 /**
  * Create flow for Deploying a database using only VS Code-native APIs such as QuickPick
  */
-export async function launchDeployDatabaseQuickpick(project: Project): Promise<IDeployProfile | undefined> {
-
-	// Show options to user for deploy to existing server or docker
-
-	const deployOption = await vscode.window.showQuickPick(
-		[constants.deployToExistingServer, constants.deployToDockerContainer],
-		{ title: constants.selectDeployOption, ignoreFocusOut: true });
-
-	// Return when user hits escape
-	if (!deployOption) {
-		return undefined;
-	}
-
-	let localDbSetting: ILocalDbSetting | undefined;
-	// Deploy to docker selected
-	if (deployOption === constants.deployToDockerContainer) {
-		let portNumber = await vscode.window.showInputBox({
-			title: constants.enterPortNumber,
-			ignoreFocusOut: true,
-			value: constants.defaultPortNumber,
-			validateInput: input => isNaN(+input) ? constants.portMustBeNumber : undefined
-		}
-		);
-
-		// Return when user hits escape
-		if (!portNumber) {
-			return undefined;
-		}
-
-		let password: string | undefined = generator.generate({
-			length: 10,
-			numbers: true,
-			symbols: true,
-			lowercase: true,
-			uppercase: true,
-			exclude: '`"\'' // Exclude the chars that cannot be included in the password. Some chars can make the command fail in the terminal
-		});
-		password = await vscode.window.showInputBox({
-			title: constants.enterPassword,
-			ignoreFocusOut: true,
-			value: password,
-			password: true
-		}
-		);
-
-		// Return when user hits escape
-		if (!password) {
-			return undefined;
-		}
-
-		localDbSetting = {
-			serverName: 'localhost',
-			userName: 'sa',
-			dbName: project.projectFileName,
-			password: password,
-			port: +portNumber,
-		};
-	}
-	let deploySettings = await getPublishDatabaseSettings(project, deployOption !== constants.deployToDockerContainer);
-
-	// Return when user hits escape
-	if (!deploySettings) {
-		return undefined;
-	}
-
-	// TODO: Ask for SQL CMD Variables or profile
-
+export async function launchDeployAppIntegrationQuickpick(project: Project): Promise<IDeployAppIntegrationProfile | undefined> {
```

Comment:
update file name to match

Reply:
I didn't change the file name to keep all the actions for deploy (deploy workspace) here as well. deploy workspace will publish db project and deploy the app